projects/phone/src/lib/masks/phone/utils/get-phone-template.ts

```diff
@@ -0,0 +1,16 @@
+import {AsYouType} from 'libphonenumber-js/core';
+
+export function getPhoneTemplate(formatter: AsYouType, value: string): string {
+    formatter.input(value.replace(/[\s\-()[\]{}]/g, ''));
```

Comment:
Could it be replaces by the following line ?

```suggestion
    formatter.input(value.replace(/\D/g, ''));
```

Reply:
no, we need "+" char

Reply:
what about this option ?

```suggestion
    formatter.input(value.replace(/[^\d\+]/g, ''));
```